docs/rules/order.md

```diff
@@ -86,7 +86,7 @@ How groups are defined, and the order to respect. `groups` must be an array of `
   // Then the rest: internal and external type
 ]
 ```
-The default value is `["builtin", "external", "parent", "sibling", "index"]`.
+The default value is `["builtin", "external", "unknown", "parent", "sibling", "index"]`.
```

Comment:
i'm concerned about changing the defaults; can you add `unknown` without changing the defaults, for now?

Reply:
I've changed this in the last commit. Could you take a look again?

Reply:
I've changed this in the last commit. Could you take a look again?